clingwrapper/src/cpp_cppyy.h

```diff
@@ -311,7 +311,7 @@ namespace Cppyy {
 
     RPY_EXPORTED
     TCppIndex_t  GetGlobalOperator(
-        TCppType_t scope, const std::string& lc, const std::string& rc, const std::string& op) { assert(0 && "GetGlobalOperator");return 0; }
+        TCppType_t scope, const std::string& lc, const std::string& rc, const std::string& op) { assert(0 && "GetGlobalOperator");return -1; }
```

Comment:
`GetGlobalOperator` is called by `BuildOperator` at https://github.com/Vipul-Cariappa/CPyCppyy-compiler-research/blob/b59c6b1846b40ba99fc2c5d82cbbfd90401d1667/src/Utility.cxx#L271-L274.

I believe it should be -1 and not 0. By looking at the logic at caller end.